ice/src/main/java/com/altinity/ice/cli/Main.java

```diff
@@ -367,6 +369,36 @@ void deleteTable(
     }
   }
 
+  @CommandLine.Command(name = "delete-file", description = "Delete file.")
+  void deleteFile(
+      @CommandLine.Option(names = "--namespace", description = "Namespace name", required = true)
+          String namespace,
+      @CommandLine.Option(names = "--table", description = "Table name", required = true)
+          String tableName,
+      @CommandLine.Option(
+              names = {"--partition"},
+              description =
+                  "JSON array of partition filters: [{\"partition_name\": \"vendorId\", \"value\": 5}]. "
```

Comment:
it would be nice if there was a way to provide a list of values without having to repeat partition name